pages/search.js

```diff
@@ -26,7 +26,41 @@ export async function getServerSideProps(context) {
     const res = await fetch(
       `${process.env.NEXT_PUBLIC_BASE_URL}/api/discover/tags`
     );
-    data.tags = await res.json();
+
+    const tags = await res.json();
+
+    /*
```

Comment:
I think the code is self explanatory in this case and perhaps we do not need the comment?  Appreciate this though! @eddiejaoude wdyt? keep the comment, or would you understand what is going on here without it?

Reply:
Yes I agree, I think the comments can be removed

Reply:
👍done